sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -652,49 +653,76 @@ def _to_internal_entity(self):
 
 
 class SqlRuleFilter(object):
-    def __init__(self, sql_expression=None):
+    def __init__(self, sql_expression=None, parameters=None, requires_preprocessing=None):
```

Comment:
docstrings?

Reply:
Added docstring to filter / action classes. (Actually copied from similar classes of ARM and updated a bit)